packages/server/src/validator/objectTypes.ts

```diff
@@ -1,109 +1,155 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
 import { Validator } from '.';
 import { predefined } from '@hashgraph/json-rpc-relay';
 
-export const OBJECTS_VALIDATIONS = {
+export type IObjectSchema = {
+  failOnEmpty?: boolean;
+  properties: {
+    [key: string]: {
+      type: string;
+      nullable: boolean;
+      required?: boolean;
+    };
+  };
+};
+
+export interface IObjectValidation {
+  validate(): boolean;
+  name(): string;
+}
```

Comment:
Let's create a new `types` folder for the `server` package and then host types and interfaces there!